app/services/solidus_bolt/transactions/capture.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Transactions
+    class Capture < SolidusBolt::Base
+      attr_reader :transaction_reference, :amount, :currency
+
+      def initialize(transaction_reference:, amount:, currency:)
+        @transaction_reference = transaction_reference
+        @amount = amount
+        @currency = currency
+        super
+      end
+
+      def call
+        capture
+      end
+
+      private
+
+      def capture
+        options = build_options
+        HTTParty.post(
+          "#{api_base_url}/#{api_version}/merchant/transactions/capture",
+          options
```

Comment:
the call works as it written like this? when I tested it the `options` needed to be separated

Reply:
Yes, it works.

I had another problem with the `BOLT_ENVIRONMENT_URL="https://api-sandbox.bolt.com"` that was set to `https://api-sandbox.bolt.com/` (with a `/` at the end), that makes the URL `https://api-sandbox.bolt.com//v1/**` and doesn't work

Reply:
I have tested it, it works like this as well